Sources/SolToBoogieTest/RegressionExecutor.cs

```diff
@@ -179,6 +196,24 @@ private string GenerateCorralArguments(CorralConfiguration corralConfig)
             return String.Join(" ", commands);
         }
 
+        private void readRecord()
+        {
+            StreamReader records = new StreamReader(Path.Combine(recordsDir, "records.txt"));
+            string line;
+            while((line = records.ReadLine()) != null)
+            {
+                string fileName = line.TrimEnd();
+                if (fileName.EndsWith('#'))

```

Comment:
I would have preferred StartsWith('#') to ignore a line. For example, this allows adding a sentence at the top of the file such as "#This is the file with "

Reply:
I see the point. Now changed:
bbb07fb900828cb4ebd2f9108f0335ff11bb1e2f